Services/Notifications/classes/class.ilNotificationGUI.php

```diff
@@ -1,13 +1,10 @@
 <?php
```

Comment:
Please add `strict_types`.

Reply:
Same here. I would outsource this into a major refactoring/PHP8 PR